javascript/deploynaut.js

```diff
@@ -79,6 +79,8 @@
 			function(data) {
 				status.text(data.status);
 				content.text(data.content);
+                //scroll the content to the bottom
```

Comment:
We-ooh-wee-ohh. The indentation police are here to tell you that these spaces should be tabs ;-)